torch/nn/_functions/dropout.py

```diff
@@ -45,7 +55,13 @@ def forward(cls, ctx, input, p=0.5, train=False, inplace=False):
 
     @staticmethod
     def backward(ctx, grad_output):
-        if ctx.p > 0 and ctx.train:
+        if hasattr(ctx, 'use_fused_kernel') and ctx.use_fused_kernel:
```

Comment:
Why would it not have this attribute?

Reply:
AlphaDropout and AlphaFeatureDropout override `forward` and thus don't set this attribute, but they share `backward` with `Dropout` class. So it's either set `use_fused_kernel` to `False` in AlphaDropout, or check for its existence in `backward`

Reply:
I'd go for setting `use_used_kernel` to `False`. `hasattr` checks are quite slow.